skyportal/handlers/api/photometry.py

```diff
@@ -1431,6 +1432,13 @@ def put(self):
                                 added in request. Can be used to later delete all
                                 points in a single request.
         """
+        user_permissions = self.current_user.permissions
```

Comment:
All `Full users` that the "Upload data" permission already, so why not just use the `permission` decorator as before?

Reply:
I think just decorating with @permissions(["Upload data") is enough

Reply:
I think that permission should always be needed, and all Full users have it anyways

Reply:
the manage photometry is enforced in the data model, just as you did.

Reply:
so users require upload photometry to use this endpoint (i.e., can they interact with photometry to edit/delete), and which data they can access is dependent on the manage photometry  (i.e. can see edit/delete THESE specific photometry points). One of them is about can they use the feature, the other is about what data points can they use the feature on.